correios/models/builders.py

```diff
@@ -112,7 +112,7 @@ def build_post_info(self, data, user: User) -> PostInfo:
         return post_info
 
     def build_receipt(self, data) -> Optional[Receipt]:
-        if (data.status_processamento == Receipt.STATUS_UNPROCESSED or data.status_processamento == ""):
+        if not data.status_processamento == Receipt.STATUS_PROCESSED:
```

Comment:
why not use `!=` here?

Reply:
I changed this. != is more efficient is better to read